.gitignore

```diff
@@ -2,6 +2,7 @@
 *.so
 
 # Distribution / packaging
+vendor/github.com/therecipe/qt/
```

Comment:
@stdevAlDen I think we need this in the gitignore, cause some of the deps in therecipe/qt are needed and static, example `widgets`. Any special reason to remove it?

Reply:
> @stdevAlDen I think we need this in the gitignore, cause some of the deps in therecipe/qt are needed and static, example `widgets`. Any special reason to remove it?

@stdevMac when therecipe qt binding is inside the vendors the project does not build (at least for me), no other thing...

Reply:
but adding it in `.gitignore` hides the fact that there is something wrong with the working copy .